client/test/components/Footer/index.test.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import Footer from '@/components/Footer';
+import { render, RenderResult } from '@testing-library/react';
+
+// TODO: 이 정규식들은 src에서도 쓰일 수 있을것 같아서 client/common/constants/regex.ts로 분리
+const PHONE_REGEX = /^[\d]+(-[\d]+){1,}$/;
+const FAX_REGEX = /^[\d]+(-[\d]+){1,}$/;
+const EMAIL_REGEX = /^[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*@[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*.[a-zA-Z]{2,3}$/i;
+
+describe('Footer', () => {
+  const renderFooter = (): RenderResult => {
+    return render(<Footer />);
+  };
+
+  const { findByAltText, container } = renderFooter();
+
+  test('render footer img', () => {
+    findByAltText('footer logo');
+  });
+
+  test('check regex phone number', () => {
+    const element = container.querySelector('[data-testid="phoneNumber"]');
+    if (element !== null) {
+      expect(element.textContent).toMatch(PHONE_REGEX);
+    }
```

Comment:
```suggestion
    const { getByTestId } = renderFooter();
    const { textContent } = getByTestId('phoneNumber');
    expect(textContent).toMatch(PHONE_REGEX);
```
`testId`를 쓰신다면 `getByTestId`를 쓰는 것도 방법이라고 생각합니당

Reply:
너무 좋습니다 :)
덕분에 getByTestid라는 함수가 있다는 사실을 알게됬네요...
정말 필요해서 찾고있었는데 ㅠㅠ 공식문서를 아직 잘 살펴보질 못했나봐요~~